Reflections/Week_6_reflection.md

```diff
@@ -0,0 +1,44 @@
+# Reflection 6
+Jose Manuel Cortes
+
+*05/014/2021*
+
+---
+
+1. What have you accomplished this week? (Please list each accomplishment, and explain)
+
+
+    * User Interface
+
+      UI has been completely switched to utilizing Grid manager within Tkinter library. Futheremore, I have reorganized the main window and placed the appropriate spacing for the graphic widget. I also allowed resizing from the x-axis and changed the resoltion to better fit all the widgets and allow for an increased ease of use. Another major change was the live-updating graphs to display temperature/humidity from the sensor. It displays data from everyweek starting on Sunday. The refresh interval refresh is user definable as to either help with performance or upon a user's preference for update. Furthermore, I added a royalty-free icon to the program and populated the Settings UI with all appropriate elements and save functionality. 
+
+    * Backend
+
+      Performed another code clean up to remove any un-necessary imports and random bits of code that performed essentially nothing. I also updated code to follow the pep8 formatting to make it more human readable. Docstrings were added to the majority of all functions as to provide a brief description and define any parameters and return's uses. I added a function to save settings from the UI menu and implemented the use of settings.json into user definable fields found within the main UI class. I also updated some naming convention within the variables to keep it more consistant and make it a less confusing experience, if code is to be revisited. Function to grab current week's sensor data was also implemented as simply query that grabs today's date and then contructs a query asking for the week's data based on today's date. More backend details found within GIT.
+
+
+
+2. What challenges or difficulties did you face? If you solved them, how? If not, what have you learned so far? Have you sought help or other resources?
```

Comment:
Great job finding an external tool to help with the threading. Don't forget to create a pull request for your development branch, you do not have to merge it until you are ready.